### PR TITLE
feat: updated datasets for dim / data_monitoring datasets and adding explores

### PR DESCRIPTION
**Pull request by kik-kik**
# feat: updated datasets for dim / data_monitoring datasets

### CODE REVIEW

File: custom-namespaces.yaml [hunk 1]
```diff
@@ -1028,16 +1028,29 @@ data_integrity_monitoring:
...
+        - table: data-monitoring-dev.dim_external.run_processing_info
       type: table_view
     muted_alerts:
       tables:
-        - table: data-monitoring-dev.dim.dim_muted_alerts
+        - table: data-monitoring-dev.dim_external.muted_alerts
       type: table_view
+  explores:
+    run_history:
+      type: explore
```
Comment (fbertsch) [inline custom-namespaces.yaml:1043]: Is this a valid explore type?

File: custom-namespaces.yaml [hunk 2]
```diff
@@ -1028,16 +1028,29 @@ data_integrity_monitoring:
   views:
     run_history:
       tables:
-        - table: data-monitoring-dev.dim.dim_run_history
+        - table: data-monitoring-dev.dim_external.run_history
       type: table_view
     run_processing_info:
       tables:
-        - table: data-monitoring-dev.dim.dim_run_processing_info
+        - table: data-monitoring-dev.dim_external.run_processing_info
```
Comment (lucia-vargas-a) [inline custom-namespaces.yaml:1035]: LGTM.
You may want to use the standard for datasets that we use in moz-fx-data-shared-prod:
e.g. dim_derived (to store tables) > dim (to store views)

**Review by lucia-vargas-a — APPROVED** [review]

File: custom-namespaces.yaml (hunk 1 above)
Comment (kik-kik) [reply custom-namespaces.yaml:1043]: It appears that the modules defined here:
https://github.com/mozilla/lookml-generator/tree/main/generator/explores

Correspond to explore types? Could be wrong here.

File: custom-namespaces.yaml (hunk 2 above)
Comment (kik-kik) [reply custom-namespaces.yaml:1035]: hmm, I thought about it but `_derived` would also not be correct in this case as this table is populated by the application itself. Could have always called it something like `dim_operational`, but we can always do that if we ever decide to invest more time into it.

File: custom-namespaces.yaml (hunk 1 above)
Comment (kik-kik) [reply custom-namespaces.yaml:1043]: I can try running generator after merge and worse case scenario remove the explores here and add them in looker-spoke

Comment (kik-kik) [reply custom-namespaces.yaml:1043]: I see, thanks for pointing this out. I'll just take it out from here and add it to looker-spoke instead.

**Review by fbertsch — APPROVED** [review]

**Pull request merged**